spring-integration-azure/src/main/java/com/microsoft/azure/spring/integration/core/Tuple.java

```diff
@@ -12,11 +12,23 @@
     private final T first;
     private final U second;
 
-    public Tuple(T first, U second) {
+    private Tuple(T first, U second) {
```

Comment:
interesting to see Java doesn't have Tuple implemented in SDK. Wanted to bring it up for discussion: from design perspective, do you think this is better https://msdn.microsoft.com/en-us/library/system.tuple(v=vs.110).aspx? e.g. make it static, have a create method, no getFirst() method etc.

Reply:
I have static factory method `of`.